packages/select/src/components/select/suggest.tsx

```diff
@@ -77,11 +77,9 @@ export class Suggest<T> extends React.PureComponent<ISuggestProps<T>, ISuggestSt
     // one, necessarily non-`static` place. note that Partial is necessary to
     // avoid having to define all required props, and all of these values must
     // be referenced manually in code.
-    private DEFAULT_PROPS: Partial<ISuggestProps<T>> = {
+    private DEFAULT_PROPS = {
```

Comment:
why remove the typedef?

Reply:
also I think you can make this `static`

Reply:
the typedef preserves the `| undefined` from optional props, which doesn't help anyone